spring-session/src/main/java/org/springframework/session/jdbc/JdbcOperationsSessionRepository.java

```diff
@@ -261,6 +261,22 @@ public void setConversionService(ConversionService conversionService) {
 		this.conversionService = conversionService;
 	}
 
+	/**
+	 * Set the {@link ClassLoader} to use for deserialization with the default
+	 * {@link ConversionService}. Applicable only with Spring 4.2.1 or later.
+	 * @param classLoader the bean class loader
+	 */
+	public void setClassLoader(ClassLoader classLoader) {
```

Comment:
User's might be a bit surprised to find out if the manually invoke `setConversionService` and then `setClassLoader` that the `ConversionService` is overridden.

Perhaps a better approach could be to remove the `setClassLoader` method and have the configuration class create a default `ConversionService`  with the appropriate `ClassLoader` if the user has not explicitly provided one.